src/main/webapp/js/script.js

```diff
@@ -76,6 +76,128 @@ function showOrHideNotes(version) {
     $('div.setup-panel div a.btn-success').trigger('click');
 });
 
+
+
+
+//Fetch data from API endpoint
+ 
+// Fetch for landing page
+
+let doms = ["web", "android", "games", "ios"]
+
+async function getPlatforms() {
+    const response = await fetch('/v1/platforms');
+    const platforms = await response.json();
+  
+    count = 0;
+    for (let plat in platforms){
+       if (plat == doms[count]){
+        const divElement = document.getElementById(plat); 
+        const textNode = document.createTextNode(platforms[plat]); 
+        divElement.appendChild(textNode);
+       }
+       count+=1;
+    }
+}
+
+//Fetch release pages
+async function getReleases(platform) {
+   const response = await fetch(`/v1/platforms/${platform}/releases`);
+   const releases = await response.json();
+   
+    //Create text elements for each release detail
+    for (i = 0; i < releases.length; i++) {
+        let element = "#release-" + i;
+        let release = releases[i];
+
+        const divElement = document.querySelector(element);
+        
+        const name = release["releaseName"];
+        const textNode0 = document.createTextNode(name); 
+        divElement.appendChild(textNode0);
+       
+        divElement.appendChild(document.createElement("HR"));
+
+        const deadline = release["launchDeadline"];
+        const textNode1 = document.createTextNode("Launch Deadline: " + getDate(1595424871)); 
+        divElement.appendChild(textNode1);
+
+        divElement.appendChild(document.createElement("HR"));
+
+        const coldfreeze = release["codeFreezeDate"];
```

Comment:
cold -> code. This is probably from the other PRs that were merged into this branch, but could you please change this.